pkg/cmd/delete/edge_application/edge_application_test.go

```diff
@@ -101,6 +101,6 @@ func TestCreate(t *testing.T) {
 		_, err := cmd.ExecuteC()
 		require.NoError(t, err)
 
-		assert.Equal(t, "Remote resources deleted successfully\n\n", stdout.String())
+		assert.Equal(t, "Remote resources deleted successfully\n", stdout.String())
```

Comment:
Use the message variable to centralize maintenance in tests

Reply:
remove \n to leave the next line formatting in the output method and be centered leaving only the message